shared/team-building/index.tsx

```diff
@@ -132,15 +138,178 @@ const ContactsBanner = ({onRedoSearch, onRedoRecs}: {onRedoSearch: () => void; o
 }
 
 class TeamBuilding extends React.PureComponent<Props, {}> {
+  sectionListRef = React.createRef<Kb.SectionList>()
   componentDidMount = () => {
     this.props.fetchUserRecs()
   }
 
+  _alphabetIndex = () => {
+    return (
+      <Kb.Box2 direction="vertical" centerChildren={true} style={styles.alphabetIndex}>
+        {this.props.recommendations &&
+          this.props.recommendations.map(section =>
+            section.label && section.label.length <= 3 ? (
+              <React.Fragment key={section.label}>
+                <Kb.Text type="BodyTiny" onClick={() => this._onScrollToSection(section.label)}>
```

Comment:
Is this going to cause unnecessary rerenders?

Reply:
Yes, I could `React.memo` this class method out, but maybe we can leave that for a second pass.